### ISSUE TITLE
[EKS] [request]: describe request here

### ISSUE THREAD

**Issue by zodiac12k** [aws/containers-roadmap#854]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Tell us about your request**
I suggest this issue #733 reopen. Because I think this issue is not resolved.
After creating the node group, running the 'kubectl get node' command gives the following result:
```
NAME                                              STATUS   ROLES        AGE     VERSION
ip-10-0-158-206.ap-northeast-2.compute.internal   Ready    <none>       8m37s   v1.15.10-eks-bac369
```

If this issue is resolved, additional configuration is not required to assign the node role and you can see the following result.
```
NAME                                              STATUS   ROLES        AGE     VERSION
ip-10-0-17-82.ap-northeast-2.compute.internal     Ready    worker       27d     v1.15.10-eks-bac369
```

**Which service(s) is this request for?**
EKS.

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
You can see the #733 issue.

**Are you currently working around this issue?**
You can see the #733 issue.

**Comment by TBBle** [comment]
I think this would be the wrong use for the node-role label. It would preclude having multiple Managed Nodegroups fulfilling the same role.

**Comment by ivan046** [comment]
> It would preclude having multiple Managed Nodegroups fulfilling the same role.
Yes! that's  the idea! we need nodes from same group to be listed with their's role via `kubectl get node`
